rholang/src/test/scala/coop/rchain/rholang/interpreter/SubstituteTest.scala

```diff
@@ -188,3 +188,38 @@ class EvalSubSpec extends FlatSpec with Matchers {
     result should be(expected)
   }
 }
+
+class BundleSubSpec extends FlatSpec with Matchers {
+  "Bundle" should "only substitute body of expression" in {
+    val source: Par = GPrivate()
+    val env         = Env.makeEnv(source)
+    val target      = Bundle(Send(ChanVar(BoundVar(0)), List(Par()), false, 0, BitSet(0)))
+    val result      = substitute(target)(env)
+    result should be(
+      Bundle(Send(Quote(source), List(Par()), false, 0, BitSet()))
+    )
+  }
+
+  it should "only substitute all Channels in body of express" in {
```

Comment:
I think that should be expression.
it should "substitute all Channels inside body"